django/control/api/router.py

```diff
@@ -2,4 +2,4 @@
 
 # Register your API views here
 
-router = routers.DefaultRouter()
+router = routers.SimpleRouter()
```

Comment:
Per documentation, the default router add an API root view that returns a response containing hyperlinks to all the list views. Wouldn't be nice to have that ? When using the django rest api ui ?

Reply:
1. It does not work with multiple routers :/
2. The OpenAPI spec will be enough for humans. Clients will be ignorant of this (especially since we're going to generate sdk, etc.).